src/tokenizer/index.ts

```diff
@@ -97,17 +93,157 @@ export default class Tokenizer {
         this.setSettings(this.settings)
         this.onLoad() // Call on load
       }
-
       // Add iframe
       this.container = el
       this.container.appendChild(this.iframe)
+      this.initApplePay(el)
     })
   }
 
   // Only here because it used originally here from the begining
   // DO NOT REMOVE, some clients still call create even though it doesnt do anything anymore.
   public create () { }
 
+  private getApiKeyURL(): string {
+    return this.getTokenizerURL() + '/' + this.apikey
+  }
+
+  private getTokenizerURL(): string {
+    const urlPrefix = this.url
+    if (window.location.href.includes('localhost')) {
+      return localDevUrl.replace(/\/$/, '') + pathUrl
+    } else {
+      return urlPrefix.replace(/\/$/, '') + pathUrl
+    }
+  }
+
+  private initApplePay(container: Element) : void {
```

Comment:
can we implement any parts from walletjs to alleviate building out the same things we already do in walletjs? just wondering if we can removed any duplicate code here.

if code size is your issue lets talk about it cause i dont know if walletjs adds enough to care.

Reply:
I like the idea, thanks!

Reply:
WalletJS is using a different approach that I don't want to follow. The only thing I was using from that tool was the structure of the settings.

Reply:
We talked about it over the phone. No longer relevant.